util/chplenv/third-party-pkgs

```diff
@@ -76,8 +73,8 @@ def compute_required_pkgs():
             if not (var.endswith('_PLATFORM') or var == 'CHPL_MEM'):
                 if val in get_all_pkgs():
```

Comment:
AFAIK we no longer need this branch of the conditional at all. Since if we're using the included 3rd-party package then val would be `bundled`, right? And besides, it would be nice not to effectively run `ls` on each line we get from printchplenv. I think you can also remove `get_all_pkgs`.

Reply:
Don't we still need to support e.g. `CHPL_LIBFABRIC=libfabric` until some time has passed?  Or does that support fall out of the other changes?  (I haven't read the logic closely yet.)

Reply:
Yes, we still support `CHPL_LIBFABRIC=libfabric`. `util/chplenv/chpl_libfabric.py` issues a deprecation warning and then converts it to `CHPL_LIBFABRIC=bundled`.

Some things still use that branch of the conditional. I see `CHPL_REGEXP=re2` and `CHPL_TARGET_MEM=jemalloc` using it in my environment.

Reply:
I think we should add those to the map you are creating in that event rather than relying on the old logic.

After that, I think we should convert these as well to use `bundled`.

Reply:
Can I create a new issue to do that? I'm worried about removing the old logic without taking the time to make sure the list of things that should be added to the map is exhaustive.

Reply:
Of course.